README.md

```diff
@@ -1,5 +1,7 @@
 # LINC Brain Client
 
+## The LINC Brain Client has been deprecated as of November 2024. Users should instead refer to the [DANDI CLI PyPI package](https://pypi.org/project/dandi/) and [DANDI CLI repository](https://github.com/dandi/dandi-cli).
```

Comment:
```suggestion
## The LINC Brain Client has been deprecated as of November 2024. Users should instead use the DANDI Client (`dandi`) with the `-i linc` argument.  See the DANDI Client [PyPI package](https://pypi.org/project/dandi/) and [GitHub repository](https://github.com/dandi/dandi-cli).  For further details, please refer to the [LINC Docs](https://docs.lincbrain.org/upload/).
```

Reply:
Did you want just https://docs.lincbrain.org/ instead of https://docs.lincbrain.org/upload?

Reply:
Thanks.  I updated my suggestion.  I would suggest the `Upload` page.